include/deal.II/base/parameter_handler.h

```diff
@@ -202,6 +301,21 @@ namespace Patterns
      */
     static std::unique_ptr<Integer> create (const std::string &description);
 
+  protected:
+    /**
+     * Convert, if possible, the content of the pointer contained in the
```

Comment:
The documentation here is saying that you should not do

~~~
int a;
boost::any t(a);
auto s = any_to_string(t);
~~~
because this won't work. It should be
~~~
int a;
boost::any t(&a); // Notice the address here
auto s = any_to_string(t);
~~~